### PR TITLE
Use arkworks for polynomails and benchmark relevant functions

### PR DESCRIPTION
**Pull request by theref**

### CODE REVIEW

File: tpke/src/refresh.rs [hunk 1]
```diff
@@ -110,6 +111,20 @@ fn make_random_polynomial_at<E: PairingEngine>(
...
+    threshold: usize,
+    root: &E::Fr,
+    rng: &mut impl RngCore,
+) -> Vec<E::Fr> {
+    let mut threshold_poly = DensePolynomial::<E::Fr>::rand(threshold - 1, rng);
+    threshold_poly[0] = E::Fr::zero();
+    let d_i_0 = E::Fr::zero() - threshold_poly.evaluate(root);
+    threshold_poly[0] = d_i_0;
+    assert_eq!(threshold_poly.evaluate(root), E::Fr::zero());
+    assert_eq!(threshold_poly.coeffs.len(), threshold);
```
[the remark this answers is not in the document]
Comment (theref) [reply tpke/src/refresh.rs:122]: awesome, didn't even know it existed, thank you!

File: tpke/benches/benchmarks.rs [hunk 2]
```diff
@@ -143,5 +148,50 @@ pub fn bench_decryption(c: &mut Criterion) {
...
 }
 
-criterion_group!(benches, bench_decryption);
+pub fn bench_random_poly(c: &mut Criterion) {
+    use rand::SeedableRng;
+    let mut group = c.benchmark_group("RandomPoly");
+    type E = ark_bls12_381::Bls12_381;
+    group.sample_size(10);
+
+    for threshold in [20, 40, 60, 80, 100] {
```
Comment (piotr-roslaniec) [inline tpke/benches/benchmarks.rs:157]:
```suggestion
    for threshold in [1, 2, 4, 8, 16, 32, 64] {
```
Nothing wrong with the values you've selected, but this suggestion is closer to the real-life parameters our users are going to use.

File: tpke/benches/benchmarks.rs [hunk 3]
```diff
@@ -143,5 +148,50 @@ pub fn bench_decryption(c: &mut Criterion) {
     }
 }
 
-criterion_group!(benches, bench_decryption);
+pub fn bench_random_poly(c: &mut Criterion) {
+    use rand::SeedableRng;
+    let mut group = c.benchmark_group("RandomPoly");
+    type E = ark_bls12_381::Bls12_381;
```
Comment (piotr-roslaniec) [inline tpke/benches/benchmarks.rs:154]: Can we declare this type on line 8, outside the function?

**Review by piotr-roslaniec — APPROVED** [review]
LGTM :tada: 
Just a few comments, good job!

**Pull request merged**